mfs/system.go

```diff
@@ -1,12 +1,6 @@
 // package mfs implements an in memory model of a mutable ipfs filesystem.
 //
-// It consists of four main structs:
-// 1) The Filesystem
```

Comment:
The the whole thing may not warrant the attention, but still - you've seen the comment on line 3 - doesn't it serve as a good enough overview sans obsolete bits?